jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -337,6 +380,55 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Get the list of terms that the user has already agreed to.
+
+      @method _getAgreements
+    */
+    _getAgreements: function() {
```

Comment:
That method is not being tested explicitly, but tests such as https://github.com/huwshimi/juju-gui/blob/c92e462e31d19abfea52c48c4a3227420f7012b2/jujugui/static/gui/src/app/components/deployment-flow/test-deployment-flow.js#L862 rely on the result of that method to pass.